docs/content/querying/lookups.md

```diff
@@ -292,3 +292,10 @@ To configure a Broker / Router / Historical / Peon to announce itself as part of
 |Property | Description | Default |
 |---------|-------------|---------|
 |`druid.lookup.tierName`| The tier for **lookups** for this node. This is independent of other tiers.|`__default`|
+
+
+## Introspect a Lookup
+
+Lookup implementations can provide some introspection capabilities by implementing `LookupIntrospectHandler`. User will send `POST` request to `/druid/lookups/v1/introspect/{lookupId}` to introspect a given Lookup.
```

Comment:
suggest `to enable introspection on a given lookup`

Also, is there a way to "un-introspect" a lookup?